subsys/usb/device/class/cdc_acm.c

```diff
@@ -538,7 +537,7 @@ static int cdc_acm_fifo_fill(const struct device *dev,
 	wrote = ring_buf_put(dev_data->tx_ringbuf, tx_data, len);
 	irq_unlock(lock);
 	if (wrote < len) {
-		LOG_WRN("Ring buffer full, drop %zd bytes", len - wrote);
+		LOG_INF("Ring buffer full, %zd bytes can't buffered just now", len - wrote);
```

Comment:
Perhaps this block,
```c
	if (wrote < len) {
		LOG_WRN("Ring buffer full, drop %zd bytes", len - wrote);
	}
```
can simply be removed or replaced with a 
`LOG_DBG("Wrote %zd bytes to TX ringbuffer", wrote);`

Reply:
I like that. 
Proposing 
`LOG_DBG("Wrote` %zd of %zd bytes to TX ringbuffer", wrote, len);`
to have the full information in DBG mode (can see if buffer is full).